terraform/deploy/locals.tf

```diff
@@ -20,6 +20,13 @@ locals {
     Team         = "DataWorks"
   }
 
+  env_prefix = {
+    management-dev = "mgt-dev."
+    management     = "mgt."
+  }
+
+  dw_domain = "${local.env_prefix[local.environment]}dataworks.dwp.gov.uk"
```

Comment:
This last bit should already be in locals

Reply:
locals is a full 37 lines long and surprisingly doesn't contain the domain name anywhere. That surprised me so I went looking and a very similar `local.parent_domain_name` ends up getting injected into terraform.tf via the bootstrap process :( That, however, puts the `wip` prefix on as well which I don't think is appropriate for the Kali boxes.

Anyway, the `dw_domain` has quickly become a standard as we've rolled the code out to all the other repos that need Kali boxes. We can raise a follow on PR to tidy up the slight duplication and work out what to do with that `wip` prefix at the same time.

Reply:
Sure. It shouldn't have WIP in it btw, one to fix another time though